app/tasks/static_vector_tile_cache_assets.py

```diff
@@ -197,8 +196,8 @@ async def _get_vector_tile_server(
 
     response = {
         "currentVersion": 10.7,
-        "name": metadata["title"],
-        "copyrightText": metadata["citation"],
+        "name": version,
```

Comment:
Is this correct? Looks to me that the version var holds the dataset version (as you'd expect). Is that really what's expected in the "name" field?

Reply:
Good catch.  This is fixed.